includes/Story_Post_Type.php

```diff
@@ -727,4 +732,59 @@ public function change_default_title( $data ) {
 		}
 		return $data;
 	}
+
+	/**
+	 * Registers the demo admin menu page.
+	 *
+	 * @return void
+	 */
+	public function add_menu_page() {
+		add_submenu_page(
+			'edit.php?post_type=' . self::POST_TYPE_SLUG,
+			__( 'Demo', 'web-stories' ),
+			__( 'Demo', 'web-stories' ),
+			'edit_posts',
+			'web-stories-demo',
+			static function() {
+				// This function is not actually used.
+				// \Google\Web_Stories\Story_Post_Type::redirect_demo() will redirect to the story editor anyway.
+				return null;
+			},
+			20
+		);
+	}
+
+	/**
+	 * Redirects the demo page to edit a new story.
+	 *
+	 * @return void
+	 */
+	public function redirect_demo() {
+		global $pagenow;
+
+		if ( 'edit.php' === $pagenow && isset( $_GET['page'] ) && 'web-stories-demo' === $_GET['page'] ) {
+			wp_safe_redirect(
+				add_query_arg(
+					[ 'post_type' => self::POST_TYPE_SLUG ],
+					admin_url( 'post-new.php?web-stories-demo' )
+				)
+			);
+			exit;
+		}
+	}
+
+	/**
+	 * Pre-fills story with demo content.
+	 *
+	 * @param string $content Default post content.
+	 *
+	 * @return string Pre-filled post content if applicable, or the default content otherwise.
+	 */
+	public function prefill_demo_content( $content ) {
+		if ( ! isset( $_GET['web-stories-demo'] ) ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
+			return $content;
+		}
+
```

Comment:
```suggestion
$file = WEBSTORIES_PLUGIN_DIR_PATH . 'includes/data/demo.json';
if ( ! is_readable( $file ) ) {
    return $content;
}
```

We should to make that file exists and is readable. If the file doesn't exist, this will fatel out.

Reply:
See this example 
https://github.com/google/web-stories-wp/blob/7a202268654e2f4636d5ece8b53efc2e04bc66bd/includes/Fonts.php#L58-L75